server/http_handler.go

```diff
@@ -147,7 +148,13 @@ func (t *tikvHandlerTool) getMvccByEncodedKey(encodedKey kv.Key) (*kvrpcpb.MvccG
 		},
 	}
 	kvResp, err := t.store.SendReq(tikv.NewBackoffer(context.Background(), 500), tikvReq, keyLocation.Region, time.Minute)
-	log.Info(string(encodedKey), keyLocation.Region, string(keyLocation.StartKey), string(keyLocation.EndKey), kvResp, err)
+	logutil.Logger(context.Background()).Info("get MVCC",
```

Comment:
get MVCC by encoded key failed

Reply:
No, it is not failed.

Reply:
I think we should remove this log. Only print the log when getting the mvcc info meets some errors.

Reply:
Done.